pull/context.go

```diff
@@ -114,15 +114,18 @@ type ReviewState string
 
 const (
 	ReviewApproved         ReviewState = "approved"
-	ReviewChangesRequested             = "changes_requested"
-	ReviewCommented                    = "commented"
-	ReviewDismissed                    = "dismissed"
-	ReviewPending                      = "pending"
+	ReviewChangesRequested ReviewState = "changes_requested"
+	ReviewCommented        ReviewState = "commented"
+	ReviewDismissed        ReviewState = "dismissed"
+	ReviewPending          ReviewState = "pending"
```

Comment:
@asvoboda sorry I asked you to change code like this in another project; I was wrong and the type declaration only carries in the specific case of using `iota`, i.e. you have to omit everything after the name for the type information of the first line to carry to the other lines.

Reply:
Not a problem!